test_configs/sockets/complete.test

```diff
@@ -350,3 +350,146 @@
 	],
 	test_flags: FT_FLAG_QUICKTEST
 },
+{
+	prov_name: sockets,
+	test_type: [
+		FT_TEST_LATENCY,
+		FT_TEST_BANDWIDTH,
+	],
+	class_function: [
+		FT_FUNC_SENDMSG,
+	],
+	ep_type: [
+		FI_EP_RDM,
+	],
+	av_type: [
+		FI_AV_TABLE,
+	],
+	comp_type: [
+		FT_COMP_QUEUE,
+		FT_COMP_CNTR,
+	],
+	mode: [
+		FT_MODE_ALL,
+	],
+	test_class: [
+		FT_CAP_MSG,
+		FT_CAP_TAGGED,
+	],
+	rx_cq_bind_flags: [
+		FI_SELECTIVE_COMPLETION,
+	],
+	rx_op_flags: [
+		FI_COMPLETION,
```

Comment:
Is it easy to add a special value that indicates zero flags?

Reply:
If this is left out, it will default to 0. Are you talking about a config file placeholder for that case?

Reply:
Yes. It seemed like the config that follows this is the same except for the setting of this flag.

Reply:
Yes. The problem is that all of the config flag settings are ORed together and used for all configs in that set. I don't know if that's really necessary for the bind/op_flags since each only has one option that we can test right now but that's the way it's set up for future development.

Reply:
Oh. Then having a special value for zero flags won't help here. The ORing behavior is needed so that we can set multiple flags at the same time. It does make testing different flags a little cumbersome.